packages/next/src/views/Document/getMetaBySegment.tsx

```diff
@@ -10,7 +10,7 @@ import { generateLivePreviewViewMetadata } from '../LivePreview/metadata.js'
 import { generateNotFoundViewMetadata } from '../NotFound/metadata.js'
 import { generateVersionViewMetadata } from '../Version/metadata.js'
 import { generateVersionsViewMetadata } from '../Versions/metadata.js'
-import { getViewsFromConfig } from './getViewsFromConfig.js'
+import { getViewsFromConfig } from './getDocumentView.js'
```

Comment:
Explanation: this file is planning to be renamed in #12777. In order maintain git history, this rename should be committed first. Since this file will undergo major changes, git won't recognize it as a rename, but a new file.